scripts/format.bat

```diff
@@ -0,0 +1,47 @@
+:: Formats specific parts of the library.
+
+:: .clang-format is automatically picked up by the npm package. 
+:: We should maybe check first if the clang-format npm package is installed at all.
+
+@ECHO OFF
+IF NOT EXIST ..\src (
+    ECHO Can not find source directory
+    EXIT
+)
+
+IF NOT EXIST ..\.clang-format (
+    ECHO Can not find source directory
+    EXIT
+)
+
+ECHO Formatting header files src\core
```

Comment:
I think this script can be improved in the funture https://stackoverflow.com/questions/33276921/batch-script-to-execute-some-commands-in-each-sub-folder#33277085. We should scan for all subdirectories instead. But this can be done in another pr.

Reply:
Thats useful! This was my first batch script ever.